runtime/bindings/python/src/compatibility/pyngraph/layout.cpp

```diff
@@ -0,0 +1,28 @@
+#include "openvino/core/layout.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
```

Comment:
Are these includes required? As I see there are no bindings for any container.

Reply:
removed.